pyproject.toml

```diff
@@ -1,13 +1,140 @@
 [build-system]
 requires = [
-    "setuptools >= 48, <60",
+    "setuptools >= 61",
     "setuptools_scm[toml] >= 4, <6",
     "setuptools_scm_git_archive",
     "milksnake",
     "wheel >= 0.29.0",
 ]
 build-backend = 'setuptools.build_meta'
 
+[project]
+name = "sourmash"
+description = "tools for comparing DNA sequences with MinHash sketches"
+readme = "README.md"
+
+authors = [
+  { name="Luiz Irber", orcid="0000-0003-4371-9659" },
+  { name="Mohamed Abuelanin", orcid="0000-0002-3419-4785" },
+  { name="Harriet Alexander", orcid="0000-0003-1308-8008" },
+  { name="Abhishek Anant", orcid="0000-0002-5751-2010" },
+  { name="Keya Barve", orcid="0000-0003-3241-2117" },
+  { name="Colton Baumler" },
+  { name="Olga Botvinnik", orcid="0000-0003-4412-7970" },
+  { name="Phillip Brooks", orcid="0000-0003-3987-244X" },
+  { name="Peter Cock", orcid="0000-0001-9513-9993" },
+  { name="Daniel Dsouza" },
```

Comment:
Hi @dsouzadaniel! Can you please provide your ORCID for our records? =]

Reply:
Hi Luiz, Here is my ORCID : 0000-0001-7843-8596
Thank you!